scripts/exgdas_atmos_analysis_wdqms.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -x
+
+# Input GSI diagnostic file containing inputs to wdqms.py
+CNVSTAT="${CDUMP}.t${cyc}z.cnvstat"
+
+# Input files from CNVSTAT fed to wdqms.py
+INPUT_LIST=("diag_conv_ps_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_t_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_q_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_uv_ges.${PDY}${cyc}.nc4")
+
+# Observation types being processed by wdqms.py
+OTYPES=(SYNOP TEMP MARINE)
+
+################################################################################
+echo "Begin job ${job:-}"
+
+#-------------------------------------------------------------------------------
+# Enter working directory
+cd "${DATA}" || ( echo "FATAL ERROR: Unable to cd ${DATA}, ABORT!"; exit 2 )
+
+#-------------------------------------------------------------------------------
+# Copy cnvstat file from COMIN to DATA, untar and gunzip input files for wdqms.py
+cp "${COMIN}/${CNVSTAT}" .
+rc=$?
+(( rc != 0 )) && ( echo "FATAL ERROR: Unable to copy ${CNVSTAT} from ${COMIN}, ABORT!"; exit 2 )
+for file in "${INPUT_LIST[@]}"; do
+  tar -xvf "${CNVSTAT}" "${file}.gz"
+  rc=$?
+  (( rc != 0 )) && ( echo "FATAL ERROR: Unable to extract ${file}.gz from ${CNVSTAT}, ABORT!"; exit 3 )
+  gunzip "${file}.gz"
+  rc=$?
+  (( rc != 0 )) && ( echo "FATAL ERROR: Unable to gunzip ${file}.gz, ABORT!"; exit 3 )
+done
+
+#-------------------------------------------------------------------------------
+# Loop over observation types and produce csv files
+for otype in "${OTYPES[@]}"; do
+  echo "Processing ... ${otype}"
+  ${WDQMSPY} -i ${INPUT_LIST[@]} -t "${otype}" -o "${DATA}"
+  rc=$?
+  if (( rc != 0 )); then
+    echo "FATAL ERROR: wdqms.py failed to process observation type ${otype}; ABORT!"
```

Comment:
@emilyhcliu 
Should this be a "FATAL ERROR"? 
If this job fails, I would think a "WARNING" should be sufficient and we continue.
Making this a "FATAL ERROR" likely raises the stakes significantly.
Please discuss this with the team.

Same question for copying them to `COMOUT` if this program succeeds in running, but no output file is created.

Reply:
In both cases, WARNING should be good.  The gdaswdqms does not affect other processes.  There is no need to stop the global-workflow.

Reply:
Ok. Yeah. 
I don't think we need to halt the GFS if this job fails.
However, we need to communicate to Ops that if this job fails, it would be good to get developer attention.

 I will update the message and error trapping.